rules/msk_topic_config_test.go

```diff
@@ -738,3 +741,12 @@ resource "kafka_topic" "good topic" {
 		})
 	}
 }
+
+func setExpectedRule(expected helper.Issues, rule *MSKTopicConfigRule) helper.Issues {
+	res := make([]*helper.Issue, len(expected))
+	for i, exp := range expected {
+		exp.Rule = rule
+		res[i] = exp
+	}
+	return res
```

Comment:
nit: this function both mutates an input (via `exp.Rule = rule`) and returns a separate output. I think it should either be pure, and just return the useful bits, or mutate its input and return nothing

Reply:
Riight.. I didn't realise I get a pointer to the issue. Will address separate